deploy/modules/gcp/bastion/bastion.tf

```diff
@@ -0,0 +1,62 @@
+resource "google_compute_firewall" "allow_ssh_bastion" {
+  name    = "allow-ssh-bastion"
+  network = var.vpc_name
+  project = var.gcp_project
+
+  allow {
+    protocol = "tcp"
+    ports    = ["22"]
+  }
+
+  source_ranges = ["0.0.0.0/0"]
+  target_tags   = ["bastion"]
+}
+
+resource "google_compute_firewall" "allow_mysql_from_bastion" {
+  name    = "allow-mysql-from-bastion"
+  network = var.vpc_name
+  project = var.gcp_project
+
+  allow {
+    protocol = "tcp"
+    ports    = ["4000"]
+  }
+
+  source_tags = ["bastion"]
+  target_tags = ["tidb"]
+}
+
+resource "google_compute_firewall" "allow_ssh_from_bastion" {
+  name    = "allow-ssh-from-bastion"
+  network = var.vpc_name
+  project = var.gcp_project
+
+  allow {
+    protocol = "tcp"
+    ports    = ["22"]
+  }
+
+  source_tags = ["bastion"]
+  target_tags = ["tidb", "tikv", "pd", "monitor"]
+}
+
+resource "google_compute_instance" "bastion" {
+  project      = var.gcp_project
+  zone         = data.google_compute_zones.available.names[0]
+  machine_type = var.bastion_instance_type
+  name         = var.bastion_name
+
+  boot_disk {
+    initialize_params {
+      image = "ubuntu-os-cloud/ubuntu-1804-lts"
+    }
+  }
+
+  network_interface {
+    subnetwork = var.public_subnet_name
+  }
+
+  tags = ["bastion"]
+
+  metadata_startup_script = "sudo apt-get install -y mysql-client && curl -s https://packagecloud.io/install/repositories/akopytov/sysbench/script.deb.sh | bash && sudo apt-get -y install sysbench"
```

Comment:
I've tried this, and found out no mysql client installed. And when I tried to execute this command manually, I got the following error:
```
centos@bastion:~$ sudo apt-get install -y mysql-client && curl -s https://packagecloud.io/install/repositories/akopytov/sysbench/script.deb.sh | bash && sudo apt-get -y install sysbench
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following package was automatically installed and is no longer required:
  grub-pc-bin
Use 'sudo apt autoremove' to remove it.
The following additional packages will be installed:
  libaio1 mysql-client-5.7 mysql-client-core-5.7 mysql-common
The following NEW packages will be installed:
  libaio1 mysql-client mysql-client-5.7 mysql-client-core-5.7 mysql-common
0 upgraded, 5 newly installed, 0 to remove and 0 not upgraded.
Need to get 9363 kB/9377 kB of archives.
After this operation, 65.3 MB of additional disk space will be used.
Ign:1 http://us-central1.gce.archive.ubuntu.com/ubuntu bionic-updates/main amd64 mysql-client-core-5.7 amd64 5.7.26-0ubuntu0.18.04.1
Ign:2 http://us-central1.gce.archive.ubuntu.com/ubuntu bionic-updates/main amd64 mysql-client-5.7 amd64 5.7.26-0ubuntu0.18.04.1
Err:1 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 mysql-client-core-5.7 amd64 5.7.26-0ubuntu0.18.04.1
  404  Not Found [IP: 35.184.34.241 80]
Ign:3 http://us-central1.gce.archive.ubuntu.com/ubuntu bionic-updates/main amd64 mysql-client all 5.7.26-0ubuntu0.18.04.1
Err:2 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 mysql-client-5.7 amd64 5.7.26-0ubuntu0.18.04.1
  404  Not Found [IP: 35.184.34.241 80]
Err:3 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 mysql-client all 5.7.26-0ubuntu0.18.04.1
  404  Not Found [IP: 35.184.34.241 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/m/mysql-5.7/mysql-client-core-5.7_5.7.26-0ubuntu0.18.04.1_amd64.deb  404  Not Found [IP: 35.184.34.241 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/m/mysql-5.7/mysql-client-5.7_5.7.26-0ubuntu0.18.04.1_amd64.deb  404  Not Found [IP: 35.184.34.241 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/m/mysql-5.7/mysql-client_5.7.26-0ubuntu0.18.04.1_all.deb  404  Not Found [IP: 35.184.34.241 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```